pkg/kubepkg/kubepkg.go

```diff
@@ -29,6 +29,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 
+	gogithub "github.com/google/go-github/v29/github"
```

Comment:
This should be in the above import group.

Reply:
Yes, fixed in https://github.com/kubernetes/release/pull/1300